src/ExpenseTracker.Core/Manager/TransactionManager.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Transactions;
+using ExpenseTracker.Core.Dto.Transaction;
+using ExpenseTracker.Core.FileManager.Interface;
+using ExpenseTracker.Core.Manager.Interface;
+using ExpenseTracker.Core.Services.Interface;
+using Microsoft.AspNetCore.Http;
+
+namespace ExpenseTracker.Core.Manager
+{
+    public class TransactionManager : ITransactionManager
+    {
+        private readonly ITransactionService _transactionService;
+        private readonly IFileManager _fileManager;
+        private string identity = "";
```

Comment:
@BirajMainali Using class level property instead of local variable is not a good practice. Consider moving it into the method below.